schemas/stimulus/stimulus.schema.tpl.json

```diff
@@ -1,4 +1,7 @@
 {
+  "_categories": [
```

Comment:
@Peyman-N when is this used? do we have a place were we ask for any type of stimulus? (I don't remember out of my head) if so @UlrikeS91 I approve this